### DISCUSSION TITLE
Pure pro got BLOCKED!

### DISCUSSION THREAD

**Discussion by TryHardJJ8** [community/community#49464] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Our school uses relay smart agent and now it is blocked if you find away around it or maybe creating something new that has Roblox that would be amazing if you can my email is <Sensitive info deleted by Staff>

![1688C9A3-5EB0-4954-A28D-4F480439ECA5](https://user-images.githubusercontent.com/126842628/223593718-1972c96f-3533-4a37-9061-7eba535940ca.jpeg)

**Comment by Akash1134** [comment]
Hi @TryHardJJ8 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by TryHardJJ8** [comment]
Thank you I appreciate that very much

**Comment by TryHardJJ8** [comment]
Ok thanks I get it now

**Comment by TryHardJJ8** [comment]
I have a question though how do you make a game website with Roblox and cod
and all that like now.gg

**Comment by oliviatheboom** [comment]
i cant get in
